backend/btrixcloud/k8sapi.py

```diff
@@ -274,8 +290,14 @@ async def get_profile_browser(self, browserid):
         )
 
     async def _patch_job(self, crawl_id, body, pluraltype="crawljobs") -> dict:
+        """patch crawl/profile job"""
+        name = f"{pluraltype[:-1]}-{crawl_id}"
```

Comment:
```suggestion
    async def _patch_job(self, obj_id, body, pluraltype="crawljobs") -> dict:
        """patch crawl/profile job"""
        name = f"{pluraltype[:-1]}-{obj_id}"
```
To help keep it a bit more generic